### PR TITLE
Fixes #4282

### LINKED ISSUE #4282: [New game]: Intellect Quest Quiz Game
### 🎮 Game Request

I would like to make a fun and interesting Quiz Game .Please assign this task to me.

### Point down the features

Game points

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Sambit-Mondal**
## PR Description 📜

Created a Intellect Quest Quiz Extension, which, takes uses the Open Trivia Database API to fetch questions, and after the user select one of the four options, the correct answer gets marked with green, and the rest gets marked with red.

Fixes #4282 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Screenshot 2024-06-05 170922

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Sambit-Mondal,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Sambit-Mondal,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Sambit-Mondal,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Comment by Sambit-Mondal** [comment]
@kunjgit I've done the changes Sir. Kindly review

File: assets/images/Screenshot 2024-06-05 170922.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-06-05 170922.png:1]: Rename the file as per contribution guidelines for naming refer contribution.md

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Sambit-Mondal** [comment]
@ishita-43 Hello, I've done the changes. Kindly check.
Thank you!

**Review by ishita-43 — APPROVED** [review]
Ready to merge!!

**Comment by kunjgit** [comment]
Thank you  @Sambit-Mondal , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**